src/paste/deploy/loadwsgi.py

```diff
@@ -284,6 +284,13 @@ def loadcontext(object_type, uri, name=None, relative_to=None, global_conf=None)
         raise LookupError("URI has no scheme: %r" % uri)
     scheme, path = uri.split(':', 1)
     scheme = scheme.lower()
+    if scheme not in _loaders:
+        if scheme.startswith('config+'):
+            entrypoints = importlib_metadata.entry_points(
+                group='paste.config_factory', name=scheme
+            )
+            entrypoint = [ep for ep in entrypoints][0]
+            _loaders[scheme] = entrypoint.load()
```

Comment:
this is not threadsafe mutating the `_loaders` global variable when loadcontext is invoked here. I think it'd be preferable to define a local `loader = _loaders.get(scheme)` and then fall into this logic `if loader is None` without mutating the global `_loaders` var.

Reply:
Done.

I don't resolve the conversation to ensure I am properly understanding what you said to me.